lib/testing_appserver/appserver_main_page.rb

```diff
@@ -24,6 +31,20 @@ def wait_to_load
       @instance.webdriver.wait_until { main_page_documents_element.present? }
     end
 
+    def main_page(selected_module)
+      instance_eval("main_page_#{selected_module}_element.click", __FILE__, __LINE__) # choose module from main page
```

Comment:
Maybe we can replace `instance_eval` with `send("main_page_#{selected_module}_element.click")`?

Not sure if this will work for `page_object` items


But maybe just replace page_object for those links on click on xpath directly, without page object

Because right now I'm not sure that we REALLY need page objects for those specific links in top toolbar

Reply:
Unfortunately `send` doesn't work

In my opinion it looks better when code follows the same style, ie with page_object